src/omega.rs

```diff
@@ -0,0 +1,67 @@
+/// Lambert W function.
+pub trait Omega
+where
+    Self: Sized,
+{
+    /// Compute the real-valued parts of the princial branch of the Lambert W function.
+    ///
+    /// This function is the inverse of the function
+    ///
+    /// ```math
+    /// f(w) = we^w
+    /// ```
+    ///
+    /// when w >= -1.
+    ///
+    /// ## Example
+    ///
+    /// ```
+    /// use special::Omega;
+    ///
+    /// assert!((1.0.lambert_w0() - 0.5671432904097838).abs() < 1e-15);
+    /// ```
+    ///
+    /// ## References
+    ///
+    /// - T. Fukushima, Precise and fast computation of Lambert W function
```

Comment:
Please use numbers here for consistency with other reference lists:

https://github.com/stainless-steel/special/blob/main/src/gamma.rs#L37

See also the indentation. Not sure if “To be submitted” is relevant. Anyway, you can just leave the formatting to me. I will make it consistent with the rest of the package.

Reply:
I removed "To be submitted" and moved the reference to the external crate into the main description.